server/queries/readSurvey.test.js

```diff
@@ -1,66 +1,95 @@
-  //     },
-  //     {
-  //       employeeId: ObjectID('507f1f77bcf86cd799439012'),
-  //       completed: false,
-  //     },
-  //     {
-  //       employeeId: ObjectID('507f1f77bcf86cd799439013'),
-  //       completed: false,
-  //     },
-  //     {
-  //       employeeId: ObjectID('507f1f77bcf86cd799439015'),
-  //       completed: false,
-  //     },
-  //     {
-  //       employeeId: ObjectID('507f1f77bcf86cd799439016'),
-  //       completed: false,
-  //     },
-  //     {
-  //       employeeId: ObjectID('507f1f77bcf86cd799439017'),
-  //       completed: false,
-  //     },
-  //   ],
-  //   questions: [
-  //     { questionId: ObjectID('707f1f87bcf86dd799439011'), position: 1 },
-  //     { questionId: ObjectID('707f1f87bcf86dd799439021'), position: 2 },
-  //     { questionId: ObjectID('707f1f87bcf86dd799439121'), position: 3 },
-  //   ],
-  //   responses: [],
-  // };
-  // expect.assertions(1);
-  // const readDocument = await readSurvey('507f1f77bcf86cd799439001');
-  // expect(readDocument).toStrictEqual(documentToFind);
+    dateCreated: 1573826615,
+    dateEdited: 1572617513,
+    dateToPublish: 1574345041,
+    datePublished: '',
+    dateToClose: 1576937061,
+    dateClosed: '',
+    anonymous: false,
+    recipients: [
+      {
+        employeeId: ObjectID('507f1f77bcf86cd799439014'),
+        completed: false,
+      },
+      {
+        employeeId: ObjectID('507f1f77bcf86cd799439012'),
+        completed: false,
+      },
+      {
+        employeeId: ObjectID('507f1f77bcf86cd799439013'),
+        completed: false,
+      },
+      {
+        employeeId: ObjectID('507f1f77bcf86cd799439015'),
+        completed: false,
+      },
+      {
+        employeeId: ObjectID('507f1f77bcf86cd799439016'),
+        completed: false,
+      },
+      {
+        employeeId: ObjectID('507f1f77bcf86cd799439017'),
+        completed: false,
+      },
+    ],
+    questions: [
+      {
+        _id: ObjectID('707f1f87bcf86dd799439011'),
+        title: 'Please describe your first week in few sentences',
+        type: 'text',
+        required: true,
+        commentEnabled: true,
+      },
+      {
+        _id: ObjectID('707f1f87bcf86dd799439021'),
+        title: 'How did you feel at work this week?',
+        type: 'multichoice',
+        required: true,
+        commentEnabled: true,
+        options: ['really bad', 'bad', 'meh', 'good', 'great'],
+      },
+      {
+        _id: ObjectID('707f1f87bcf86dd799439121'),
+        title: "I see myself still working at B*amery in two years' time",
+        type: 'multichoice',
+        required: true,
+        commentEnabled: true,
+        options: [
+          'strongly disagree',
+          'disagree',
+          'neutral',
+          'agree',
+          'strongly agree',
+        ],
+      },
+    ],
+    responses: [],
+  };
+  expect.assertions(1);
+  const readDocument = await readSurvey('508f1f99bcf86cd799439014');
+  expect(readDocument).toStrictEqual(documentToFind);
```

Comment:
can we make an assertion on the status code please? Not major here as this assertion looks pretty solid.

Reply:
will do in a future iteration, just wanted to get some tests in for now